### ISSUE TITLE
Pretty much all duplicates

### ISSUE THREAD

**Issue by zakmayfield** [HashLips/generative-art-node#99]
Pretty much all of my builds are duplicates. Not sure how to proceed. Any suggestions?

Decoded hash gets repeated in the metada.json file like so:
`0, 1, 2 -> 0, 1, 2, 0, 1, 2 -> 0, 1, 2, 0, 1, 2, 0, 2`

<img width="238" alt="Screen Shot 2021-10-30 at 9 45 53 PM" src="https://user-images.githubusercontent.com/55856955/139563286-103875f7-2198-49d7-9f6f-f83550d86136.png">

<img width="250" alt="Screen Shot 2021-10-30 at 9 46 38 PM" src="https://user-images.githubusercontent.com/55856955/139563299-9705bfd8-c9e9-4f0b-b845-7f1b58799b91.png">

**Comment by Josh27t** [comment]
Hi, same with me. Its current correcting system for duplicates only detect it once and the newly generated image is not being scanned by the 'for loop' the second time and thus very huge potential for dupes to be formed.

**Comment by justinbalaguer** [comment]
yup same with me full of dupes. the logic for this must be changed
